DuckDuckGo/Onboarding/ContextualOnboarding/OnboardingSuggestedSearchesProvider.swift

```diff
@@ -68,7 +68,13 @@ struct OnboardingSuggestedSearchesProvider: OnboardingSuggestionsItemsProviding
 
     private var surpriseMe: ContextualOnboardingListItem {
         var search = UserText.ContextualOnboarding.tryASearchOptionSurpriseMe
-        return ContextualOnboardingListItem.surprise(title: search, visibleTitle: UserText.ContextualOnboarding.tryASearchOptionSurpriseMeTitle)
+        if var url = URL.makeSearchUrl(from: search) {
```

Comment:
Just a tought: maybe we could make an extension on URL callled `makeImageSearchUrl(from:)`? It may be useful in other scenarios and it would make the code cleaner here.